CMakeLists.txt

```diff
@@ -51,13 +51,35 @@ set(SRC_LIST
     src/Player.cpp
     src/Property.cpp
     src/Property.hpp
+    src/Cards.cpp
+    src/Cards.hpp
     src/Dice.hpp
-    src/Dice.cpp  
+    src/Dice.cpp
+    src/Field.hpp
+    src/Field.cpp
+    src/OrdinaryCard.hpp
+    src/OrdinaryCard.cpp
+    src/Railway.hpp
```

Comment:
Why so many types of cards? They are not needed now. They should be implemented later on requirements like:
```
Każdorazowe przejście przez pole "Start" lub zatrzymanie się na nim, nagradzane jest wypłacaną przez bank sumą w wysokości 400$, o ile grający nie "udaje się do wiezienia".
Pole "Parking strzeżony" traktowane jest jako pole dodatkowe nie podlegające wykupowi. Osoba znajdująca się swoim pionkiem na tym polu płaci do banku 400$. Pole "Podatek od wzbogacenia" traktowane jest podobnie, przy czym grający wpłaca do banku 200$.
Pole "Darmowy parking" funkcjonuje w grze jako pole nie przynoszące ani zysku ani strat.
```

A lot of code can be deleted now :)